test/e2e/util.go

```diff
@@ -1142,7 +1142,11 @@ func SSH(cmd, host, provider string) (string, string, int, error) {
 		return "", "", 0, fmt.Errorf("error getting signer for provider %s: '%v'", provider, err)
 	}
 
-	return util.RunSSHCommand(cmd, host, signer)
+	user := os.Getenv("KUBE_SSH_USER")
+	if user == "" {
```

Comment:
util is supposed to be pretty generic I thought, so my feeling was that it shouldn't really be looking at the environment, and it would be even more questionable to have this if statement in there.  I'm happy to just move this into RunSSHCommand (it would simplify the code) if you disagree.